build.cmd

```diff
@@ -142,6 +142,8 @@ set __RestoreOptData=1
 set __GenerateLayout=0
 set __CrossgenAltJit=
 
+set __Use64BitHostTools=32
```

Comment:
Seems awkward that something named `__Use64BitHostTools` can be either 32 or 64; seems like something with that name would be `false` or `true` (or 0 or 1).